paymentsheet/src/main/java/com/stripe/android/link/LinkControllerViewModel.kt

```diff
@@ -170,6 +170,12 @@ internal class LinkControllerViewModel @Inject constructor(
                     val state = _state.value
                     val linkAccountInfo = getLinkAccountInfo(email, state)
 
+                    if (linkAccountInfo.account?.isVerified == true) {
+                        logger.debug("$tag: account is already verified, skipping authentication")
+                        _presentForAuthenticationResultFlow.emit(PresentForAuthenticationResult.Success)
```

Comment:
we can potentially return an `AlreadyAuthenticated` error here, but might that might be more confusing for integrators than just succeed.

Reply:
agreed